ui/clubhouse.js

```diff
@@ -1266,6 +1266,44 @@ function showOverview() {
         mapNotifyWindow(notify, notify.get_compositor_private());
 }
 
+function onOnboardingHLChanged(isOnboardingHL) {
+    const app = Utils.getClubhouseApp();
+    const notify = app.get_windows().find(w => w.get_role() === 'clubhouse-msg-notify');
+
+    if (!isOnboardingHL && !Main.overview.visible) {
+        _destroyNotifyClone();
+        return;
+    }
+
+    if (notify)
+        mapNotifyWindow(notify, notify.get_compositor_private(), isOnboardingHL);
+}
+
+async function connectOnboarding() {
+    try {
+        ONBOARDING_PROXY = await Gio.DBusProxy.new(
+            Gio.DBus.session,
+            Gio.DBusProxyFlags.DO_NOT_AUTO_START,
+            null,
+            'com.endlessm.onboarding',
+            '/com/endlessm/onboarding',
+            'com.endlessm.onboarding',
+            null);
```

Comment:
I think you should pass a cancellable here and cancel it on disable()... In that case also worth to check if `error == CANCELLED` below to avoid a warning when cancelled before the proxy is ready due to the extension getting disabled.

Reply:
Done. I've added the cancellable here and cancel on disable.